phy/cluster/default_settings.py

```diff
@@ -7,47 +7,48 @@
 # Spike detection
 # -----------------------------------------------------------------------------
 
-def spikedetekt_params(sample_rate):
-
-    assert sample_rate > 0
+traces = {
+    'raw_data_files': [],
+    'n_channels': None,
+    'dtype': None,
+    'sample_rate': None,
+}
 
-    return dict(
-        # Filter.
-        filter_low=500.,
-        filter_high=0.95 * .5 * sample_rate,
-        filter_butter_order=3,
 
-        # Data chunks.
-        chunk_size=int(1. * sample_rate),
-        chunk_overlap=int(.015 * sample_rate),
+spikedetekt = {
+    'filter_low': 500.,
+    'filter_high_factor': 0.95 * .5,  # will be multiplied by the sample rate
```

Comment:
I don't know under what situation you would want to change that filter.
Might have been a parameter for a good reason before, though. Ask Kenneth?

On Wed, Jul 1, 2015 at 6:18 PM, nippoo <notifications@github.com> wrote:

> In phy/cluster/default_settings.py
> <https://github.com/kwikteam/phy/pull/448#discussion_r33701198>:
>
> >
> > -        # Data chunks.
> > -        chunk_size=int(1. * sample_rate),
> > -        chunk_overlap=int(.015 * sample_rate),
> > +spikedetekt = {
> > +    'filter_low': 500.,
> > +    'filter_high_factor': 0.95 * .5,  # will be multiplied by the sample rate
>
> Aha, now I understand. This is simply an anti-aliasing filter.
>
> Could we instead rename this parameter to aa_nyquist_filter_factor to make
> its purpose more explicit? Confusingly it's a lowpass filter.
>
> In fact I would personally vote to have this as a fixed frequency
> (0.47*sample_rate) and just set one variable aa_filter = True/False - why
> would we really want to change this? @nsteinme
> <https://github.com/nsteinme>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kwikteam/phy/pull/448/files#r33701198>.
>